### ISSUE TITLE
[Bug] - All posts return same title/metaDesc

### ISSUE THREAD

**Issue by shawngrona** [ashhitch/wp-graphql-yoast-seo#29]
Steps to repro:
1. wordpress site with yoast 14.0.3
2. install wordpress plugin wpgraphql 0.8.4
3. install wordpress plugin wp-graphql-yoasy-seo 3.3.0
4. using some graphql editor (eg GraphIQL) execute the following query:
```
{
  posts(first: 10) {
    edges {
      node {
        id
        title
        seo {
          title
          metaDesc
          focuskw
          metaKeywords
          metaRobotsNoindex
          metaRobotsNofollow
          opengraphTitle
          opengraphDescription
          opengraphImage {
            altText
            sourceUrl
            srcSet
          }
          twitterTitle
          twitterDescription
          twitterImage {
            altText
            sourceUrl
            srcSet
        }
        }
      }
    }
  }
}
```

expected:
Title and meta description reflect each individual post

actual:
Title and meta description match the most recently generated post

**Comment by ashhitch** [comment]
Hey, thanks reporting this can I ask which version of Yoast you are using?

**Comment by ashhitch** [comment]
@shawngrona I have just installed the RC of Yoast 14 and can see it has breaking changes for my plugin., I will start to look into this., if you were to install version 13 of the Yoast plugin it should work for you.

**Comment by shawngrona** [comment]
thank you @ashhitch 
i didnt think to add the yoast version above - you're right i'm on yoast 14.0.3 looks like 14.0.4 is available

**Comment by ashhitch** [comment]
@shawngrona I have a test version that should be good to go with Yoast 14 If you fancy testing it:
https://github.com/ashhitch/wp-graphql-yoast-seo/releases/tag/v4.0.0Alpha

**Comment by shawngrona** [comment]
@ashhitch thank you so much for quick turnaround, your yoast-14 branch has resolved my issue with title/metaDesc

**Comment by ashhitch** [comment]
Fixed by #31
